.gitignore

```diff
@@ -16,4 +16,8 @@
 *.idb
 *.hlsl.h
 *.bat
+*.vcxproj.user 
 imgui.ini
+.ignore/*
+/src/rt64lib/x64
+/src/sample/x64
```

Comment:
Which are the file extensions not being ignored in the x64 folders? It'd be preferable to ignore those than make it per project if possible.